### PR TITLE
Setup Activity

### PR DESCRIPTION
**Pull request by martinmona**
## Summary
- Created index.html
- Implemented Sass
- Added .gitignore

## Screenshot
image

## Trello Card
https://trello.com/c/bCvGuXSG/11-setup-del-proyecto

### CODE REVIEW

File: Maquetado/Setup Proyecto/index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+
```
Comment (manueltuero) [inline Maquetado/Setup Proyecto/index.html]: Remove empty line

File: Maquetado/Setup Proyecto/.gitignore [hunk 2]
```diff
@@ -0,0 +1,10 @@
+### Sass ###
```
Comment (manueltuero) [inline Maquetado/Setup Proyecto/.gitignore]: I think that **Setup Proyecto** folder is unnecessary. Please put the files directly inside of Maquetado folder.

File: Maquetado/.gitignore [hunk 3]
```diff
@@ -0,0 +1,10 @@
+### Sass ###
+.sass-cache/
+.scss-cache/
+*.css.map
+*.sass.map
+*.scss.map
+*.css
+
+## System ##
+.DS_Store
```
Comment (henryzarza) [inline Maquetado/.gitignore]: Add end of file (empty line)

File: Maquetado/index.html [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body>
+  <h1 class="title">Hello World</h1> 
+</body>
+</html>
```
Comment (henryzarza) [inline Maquetado/index.html]: The same here

File: Maquetado/scss/application.scss [hunk 5]
```diff
@@ -0,0 +1,3 @@
+.title {
+    color: red;
+  }
```
Comment (henryzarza) [inline Maquetado/scss/application.scss]: Add EOF (End Of File) and remove the tabulation
```suggestion
}
```

File: Maquetado/scss/application.scss [hunk 6]
```diff
@@ -0,0 +1,3 @@
+.title {
+    color: red;
```
Comment (henryzarza) [inline Maquetado/scss/application.scss]: Remember that indentation should be of 2 spaces each. Check this [style guide](https://github.com/Wolox/tech-guides/blob/master/frontend/docs/css-style-guide.md#indentation)

**Pull request merged**